build.sbt

```diff
@@ -11,7 +11,7 @@ scalaVersion := "2.10.4"
 scalacOptions += "-feature"
 
 libraryDependencies ++= Seq(
-  "org.webjars" % "rjs" % "2.1.15"
+  "org.webjars" % "requirejs-node" % "2.1.17"
```

Comment:
If you want to update this PR to use `"org.webjars.npm" % "requirejs" % "2.2.0"` I will merge it. (can you squash the commits after you do that?)